lib/bindings/MQTTBinding.js

```diff
@@ -98,9 +95,18 @@ function sendConfigurationToDevice(apiKey, deviceId, results, callback) {
     config.getLogger().debug(context, 'Sending requested configuration to the device:\n %j', configurations);
 
     mqttClient.publish(
-        '/' + apiKey + '/' + deviceId + '/' + constants.CONFIGURATION_SUFIX + '/' +
-        constants.CONFIGURATION_VALUES_SUFIX,
-        JSON.stringify(configurations), options, callback);
+        '/' +
```

Comment:
Topic/URL composition is better in just one line. Could this change be avoided?

Reply:
Fixed 7657825 - Note the line is over 120 chars unless split once anyway.